document-generator-api/pom.xml

```diff
@@ -81,7 +85,7 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.0</version>
+            <version>${gson.version}</version>
```

Comment:
This dependency is now a duplicate of the one above change it back or remove one.